### PR TITLE
Irminger 8 - Glider cals

### PR DESCRIPTION
**Pull request by cdobs**
Calibration csvs related to instruments deployed on Irminger 8 gliders and 2 additional NUTNRM cal files

### CODE REVIEW

File: calibration/PARADM/CGINS-PARADM-50198__20190925.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50198,CC_bsipar_par_scaling,1,
```
Comment (rtravis-whoi) [inline calibration/PARADM/CGINS-PARADM-50198__20190925.csv:2]: looks good

**Review by rtravis-whoi — CHANGES_REQUESTED** [review]
Just one thing to check - for CTDGVM-09237 the cal date on the actual certificate is 2021-03-04 (2021-03-24 is the date on the service report). Also - should there be 'post-cruise' cal certs like the other glider CTD cal pdfs have? The svc report says that post-cruise cal was performed.

File: calibration/CTDGVM/CGINS-CTDGVM-09237__20210304.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9237,CC_calibration_date,1614816000000,sensor calibration date (milliseconds since 1970-01-01)
```
Comment (rtravis-whoi) [inline calibration/CTDGVM/CGINS-CTDGVM-09237__20210304.csv:2]: update looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00709__20210305.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
... 272.36, 273.18, 274, 274.82, 275.64, 276.45, 277.27, 278.09, 278.91, 279.73, 280.55, 281.36, 282.18, 283, 283.82, 284.64, 285.46, 286.28, 287.09, 287.91, 288.73, 289.55, 290.37, 291.19, 292, 292.82, 293.64, 294.46, 295.28, 296.1, 296.92, 297.73, 298.55, 299.37, 300.19, 301.01, 301.83, 302.64, 303.46, 304.28, 305.1, 305.92, 306.73, 307.55, 308.37, 309.19, 310.01, 310.82, 311.64, 312.46, 313.28, 314.09, 314.91, 315.73, 316.55, 317.36, 318.18, 319, 319.82, 320.63, 321.45, 322.27, 323.08, 323.9, 324.72, 325.53, 326.35, 327.17, 327.98, 328.8, 329.61, 330.43, 331.25, 332.06, 332.88, 333.69, 334.51, 335.32, 336.14, 336.95, 337.77, 338.58, 339.4, 340.21, 341.03, 341.84, 342.66, 343.47, 344.28, 345.1, 345.91, 346.72, 347.54, 348.35, 349.16, 349.98, 350.79, 351.6, 352.42, 353.23, 354.04, 354.85, 355.66, 356.48, 357.29, 358.1, 358.91, 359.72, 360.53, 361.34, 362.15, 362.96, 363.77, 364.58, 365.39, 366.2, 367.01, 367.82, 368.63, 369.44, 370.25, 371.06, 371.87, 372.67, 373.48, 374.29, 375.1, 375.9, 376.71, 377.52, 378.33, 379.13, 379.94, 380.74, 381.55, 382.36, 383.16, 383.97, 384.77, 385.58, 386.38, 387.18, 387.99, 388.79, 389.6, 390.4, 391.2, 392.01, 392.81, 393.61, 394.41, 395.21, 396.02]",
```
Comment (rtravis-whoi) [inline calibration/NUTNRM/CGINS-NUTNRM-00709__20210305.csv:8]: looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00730__20210301.csv [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
...85, 272.65, 273.46, 274.26, 275.07, 275.87, 276.68, 277.48, 278.29, 279.09, 279.9, 280.71, 281.51, 282.32, 283.12, 283.93, 284.73, 285.54, 286.35, 287.15, 287.96, 288.76, 289.57, 290.37, 291.18, 291.99, 292.79, 293.6, 294.4, 295.21, 296.02, 296.82, 297.63, 298.43, 299.24, 300.04, 300.85, 301.66, 302.46, 303.27, 304.07, 304.88, 305.68, 306.49, 307.3, 308.1, 308.91, 309.71, 310.52, 311.32, 312.13, 312.93, 313.74, 314.54, 315.35, 316.15, 316.96, 317.76, 318.57, 319.37, 320.18, 320.98, 321.79, 322.59, 323.4, 324.2, 325.01, 325.81, 326.61, 327.42, 328.22, 329.03, 329.83, 330.63, 331.44, 332.24, 333.05, 333.85, 334.65, 335.46, 336.26, 337.06, 337.86, 338.67, 339.47, 340.27, 341.08, 341.88, 342.68, 343.48, 344.28, 345.09, 345.89, 346.69, 347.49, 348.29, 349.09, 349.9, 350.7, 351.5, 352.3, 353.1, 353.9, 354.7, 355.5, 356.3, 357.1, 357.9, 358.7, 359.5, 360.3, 361.1, 361.9, 362.7, 363.5, 364.29, 365.09, 365.89, 366.69, 367.49, 368.29, 369.08, 369.88, 370.68, 371.48, 372.27, 373.07, 373.87, 374.66, 375.46, 376.25, 377.05, 377.85, 378.64, 379.44, 380.23, 381.03, 381.82, 382.61, 383.41, 384.2, 385, 385.79, 386.58, 387.38, 388.17, 388.96, 389.76, 390.55, 391.34, 392.13, 392.92, 393.71, 394.51]",
```
Comment (rtravis-whoi) [inline calibration/NUTNRM/CGINS-NUTNRM-00730__20210301.csv:8]: looks good

File: calibration/NUTNRM/CGINS-NUTNRM-00730__20210806.csv [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
...85, 272.65, 273.46, 274.26, 275.07, 275.87, 276.68, 277.48, 278.29, 279.09, 279.9, 280.71, 281.51, 282.32, 283.12, 283.93, 284.73, 285.54, 286.35, 287.15, 287.96, 288.76, 289.57, 290.37, 291.18, 291.99, 292.79, 293.6, 294.4, 295.21, 296.02, 296.82, 297.63, 298.43, 299.24, 300.04, 300.85, 301.66, 302.46, 303.27, 304.07, 304.88, 305.68, 306.49, 307.3, 308.1, 308.91, 309.71, 310.52, 311.32, 312.13, 312.93, 313.74, 314.54, 315.35, 316.15, 316.96, 317.76, 318.57, 319.37, 320.18, 320.98, 321.79, 322.59, 323.4, 324.2, 325.01, 325.81, 326.61, 327.42, 328.22, 329.03, 329.83, 330.63, 331.44, 332.24, 333.05, 333.85, 334.65, 335.46, 336.26, 337.06, 337.86, 338.67, 339.47, 340.27, 341.08, 341.88, 342.68, 343.48, 344.28, 345.09, 345.89, 346.69, 347.49, 348.29, 349.09, 349.9, 350.7, 351.5, 352.3, 353.1, 353.9, 354.7, 355.5, 356.3, 357.1, 357.9, 358.7, 359.5, 360.3, 361.1, 361.9, 362.7, 363.5, 364.29, 365.09, 365.89, 366.69, 367.49, 368.29, 369.08, 369.88, 370.68, 371.48, 372.27, 373.07, 373.87, 374.66, 375.46, 376.25, 377.05, 377.85, 378.64, 379.44, 380.23, 381.03, 381.82, 382.61, 383.41, 384.2, 385, 385.79, 386.58, 387.38, 388.17, 388.96, 389.76, 390.55, 391.34, 392.13, 392.92, 393.71, 394.51]",
```
Comment (rtravis-whoi) [inline calibration/NUTNRM/CGINS-NUTNRM-00730__20210806.csv:8]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: calibration/CTDGVM/CGINS-CTDGVM-09237__20210304.csv (hunk 2 above)
Comment (snwhite) [reply calibration/CTDGVM/CGINS-CTDGVM-09237__20210304.csv:2]: Looks good.

File: calibration/CTDGVM/CGINS-CTDGVM-09285__20190927.csv [hunk 6]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9285,CC_calibration_date,1569542400000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/CTDGVM/CGINS-CTDGVM-09285__20190927.csv:2]: Looks good.

File: calibration/DOSTAM/CGINS-DOSTAM-00142__20201210.csv [hunk 7]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+142,CC_calibration_date,1607558400000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/DOSTAM/CGINS-DOSTAM-00142__20201210.csv:2]: Looks good.

File: calibration/DOSTAM/CGINS-DOSTAM-00168__20201210.csv [hunk 8]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+168,CC_calibration_date,1607558400000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/DOSTAM/CGINS-DOSTAM-00168__20201210.csv:2]: Looks good.

File: calibration/DOSTAM/CGINS-DOSTAM-00401__20210405.csv [hunk 9]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+401,CC_calibration_date,1617580800000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/DOSTAM/CGINS-DOSTAM-00401__20210405.csv:2]: Looks good.

File: calibration/FLORDM/CGINS-FLORDM-03528__20200423.csv [hunk 10]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+3528,CC_angular_resolution,1.096,
+3528,CC_depolarization_ratio,0.039,
+3528,CC_measurement_wavelength,700.0,
+3528,CC_scattering_angle,140.0,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/FLORDM/CGINS-FLORDM-03528__20200423.csv:5]: Looks good.

File: calibration/FLORDM/CGINS-FLORDM-03582__20210315.csv [hunk 11]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+3582,CC_angular_resolution,1.096,
+3582,CC_depolarization_ratio,0.039,
+3582,CC_measurement_wavelength,700.0,
+3582,CC_scattering_angle,140.0,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/FLORDM/CGINS-FLORDM-03582__20210315.csv:5]: Looks good.

File: calibration/FLORTM/CGINS-FLORTM-03813__20190911.csv [hunk 12]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+3813,CC_angular_resolution,1.076,
+3813,CC_depolarization_ratio,0.039,
+3813,CC_measurement_wavelength,700,
+3813,CC_scattering_angle,124,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/FLORTM/CGINS-FLORTM-03813__20190911.csv:5]: Looks good.

File: calibration/FLORTO/CGINS-FLORTO-01330__20190913.csv [hunk 13]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+1330,CC_angular_resolution,1.076,
+1330,CC_depolarization_ratio,0.039,
+1330,CC_measurement_wavelength,700,
+1330,CC_scattering_angle,124,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/FLORTO/CGINS-FLORTO-01330__20190913.csv:5]: Looks good.

File: calibration/PARADM/CGINS-PARADM-50198__20190925.csv (hunk 1 above)
Comment (snwhite) [reply calibration/PARADM/CGINS-PARADM-50198__20190925.csv:2]: Looks good.

File: calibration/NUTNRM/CGINS-NUTNRM-00709__20210305.csv (hunk 3 above)
Comment (snwhite) [reply calibration/NUTNRM/CGINS-NUTNRM-00709__20210305.csv:8]: Spot check looks good.

File: calibration/NUTNRM/CGINS-NUTNRM-00730__20210301.csv (hunk 4 above)
Comment (snwhite) [reply calibration/NUTNRM/CGINS-NUTNRM-00730__20210301.csv:8]: Spot check looks good.

File: calibration/NUTNRM/CGINS-NUTNRM-00730__20210806.csv (hunk 5 above)
Comment (snwhite) [reply calibration/NUTNRM/CGINS-NUTNRM-00730__20210806.csv:8]: Spot check looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**